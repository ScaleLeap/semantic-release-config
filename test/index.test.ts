test('should pass', () => {
  expect(true).toBeTruthy()
})
