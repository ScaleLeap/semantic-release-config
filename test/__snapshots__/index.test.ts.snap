// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should pass 1`] = `
Object {
  "plugins": Array [
    "@semantic-release/commit-analyzer",
    "@semantic-release/release-notes-generator",
    Array [
      "@semantic-release/changelog",
      Object {
        "changelogTitle": "# Changelog

          All notable changes to this project will be documented in this file.
          See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.",
      },
    ],
    Array [
      "@semantic-release/npm",
      Object {
        "tarballDir": "pack",
      },
    ],
    Array [
      "@semantic-release/github",
      Object {
        "assets": "pack/*.tgz",
      },
    ],
    "@semantic-release/git",
    Array [
      "semantic-release-slack-bot",
      Object {
        "notifyOnFail": true,
        "notifyOnSuccess": true,
      },
    ],
  ],
}
`;
